index.html

```diff
@@ -427,10 +427,225 @@ <h1 id="detail-title">&nbsp;</h1>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Recommended Pre-Requisite: </p><p>Using APIs (Intro to REST), Intro to Python</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Date & Time: </p><p>10/27/2016 1:00 - 4:00PM ET</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Instructor: </p><p>General Assembly</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Software Needed: </p><p>Python, Express.js, Mongoose, MongoDB</p></span>
+
+                                    <p class="buttons">
+                                        <a class="btn btn-primary classRegistrationButton" href="https://goo.gl/forms/HN73ecEwHvaOkAkY2" target="_blank"><i class="fa fa-globe"></i> Register</a>
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="reference-item" data-category="datascience">
+                              <div class="reference">
+                                  <a href="#">
+                                    <img src="Mixit_Panels/11-8-16.png" class="img-responsive" alt="" />
+
+                                </a>
+
+                                 <!-- Everything in this  <div> is essentially the modal -->
+                                <div class="sr-only reference-description">
+                                    <h3 class="reference-title">Excel with Excel</h3> 
+
+                                    <p class="courseDescription">Learn the basics and some more advanced aspects of Excel including worksheet management, formatting, filtering, calculations, functions, string manipulation, pivot tables, charts, and macros.</p>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Topics to Be Covered: Worksheet Management, Formatting, Filtering, Calculating, Functions, Pivot Tables, Charts, Macros</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Recommended Pre-Requisite: </p><p>None</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Date & Time: </p><p>11/8/2016 1:00 - 4:00PM ET</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Instructor: </p><p>General Assembly</p></span>
+
+                                    <span class="courseSpanTag"><p class="courseListingKeys">Software Needed: </p><p>Microsoft Office Excel</p></span>
+                               <!--Greyed out until registration form for November courses is available
```

Comment:
Since all the other classes have a register button and then only excel with excel & intro to data analysis dont, I think you should add "registration coming soon" text